app/api/v0/proofs/proved/route.ts

```diff
@@ -167,13 +171,14 @@ export const POST = withAuth(async ({ request, client, user, timestamp }) => {
     .select("proof_id")
     .single()
 
-  if (proofResponse.error) {
+  if (proofResponse.error || proofBinaryResponse.error) {
     console.error("error adding proof", proofResponse.error)
     return new Response("Internal server error", { status: 500 })
   }
 
-  // invalidate proofs cache
+  // invalidate proofs and proof_binaries cache
   revalidateTag("proofs")
+  revalidateTag("proof_binaries")
```

Comment:
There is no tag defined for `proof_binaries`, so this will do nothing. The only query that uses a tag is the home page one that uses "proofs". In this case, that would be enough.